dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops

```diff
@@ -229,3 +229,43 @@ LOGINPROPERTY_OVERRIDE_USERFILTER.useraction=If the userFilter attribute is pref
 INVALID_LOGIN_PROPERTIES=CWIML4506E: The following login properties are not valid: {0}.
 INVALID_LOGIN_PROPERTIES.explanation=One or more login properties are not valid WIM PersonAccount properties.
 INVALID_LOGIN_PROPERTIES.useraction=Either choose valid PersonAccount properties or add the properties to PersonAccount as extended properties. Ensure that the case for each login property matches the case for the corresponding PersonAccount property.
+
+KRB5_LOGIN_FAILED_CACHE=CWIML4507E: Kerberos login failed using Kerberos principal {0} and Kerberos credential cache (ccache) {1}.
```

Comment:
Kerberos login failed with the {0}  Kerberos principal and the {1} Kerberos credential cache (ccache).